### DISCUSSION TITLE
No files on device - Latest Firmware update rp2-pico-20230421-unstable-v1.19.1-1017-gbf3eb9dc3.uf2

### DISCUSSION THREAD

**Discussion by FlydGitHub** [micropython/micropython#11312] [RP2040 / Pico]
Did do something wrong?

I just installed the current update - where are my existing files?

I did't 'nuke' it............

**Comment by FlydGitHub** [comment]
Fixed in newer build - thanks

**Comment by FlydGitHub** [comment]
Still broken - for Pico.

Files show up while using Pico W version on the Pico (non W).
I accidentally installed the W build on the the base Pico

**Reply by peterhinch** [reply]
I guess the W build has a different flash layout. I think it's safest to nuke it and install the correct firmware.

**Comment by robert-hh** [comment]
The firmware size of the PICO W is much larger that that of the standard Pico. So it overwrites parts of the standard pico's file system, especially the root records.  So the data in the file system is lost. The file system location differs between boards and starts with a certain gap behind the firmware. Nuking the board as @peterhinch suggests will give you a clean restart.
